velox/functions/sparksql/tests/RegexFunctionsTest.cpp

```diff
@@ -41,8 +43,115 @@ class RegexFunctionsTest : public test::SparkFunctionBaseTest {
     return evaluateOnce<std::string>(
         fmt::format("regexp_extract(c0, '{}')", pattern), str);
   }
+
+  void regex_replace_constant(
```

Comment:
For `regex_replace` test, you can refer the one of [regex_extract](https://github.com/facebookincubator/velox/blob/main/velox/functions/lib/tests/Re2FunctionsTest.cpp#L615-L781) to cover pattern, position and int32_t/int64_t scenarios.